### DISCUSSION TITLE
Does ArgoCD need backup and restore in DR?

### DISCUSSION THREAD

**Discussion by laptua** [argoproj/argo-cd#13126] [Q&A]
I am new to ArgoCD and read the operational manual. There is a DR about backup and restore.
https://argo-cd.readthedocs.io/en/stable/operator-manual/disaster_recovery/

I would like to know regular backup is necessary or not. It is because everything in in Git repository, the state of Argo could recreated as long as Argo could access Git.

Any use case that I need to backup regularly ?

**Comment by jgwest** [comment]
The backup utility is essentially a convenient way to export the Argo CD K8s resources that are defined in the Argo CD namespace. It will output a backup containing the following resources from your Argo CD namespace:
- Various ConfigMaps
    - Argo CD settings ConfigMaps (various settings)
    - Argo CD RBAC ConfigMaps
    - KnownHosts ConfigMap
    - TLSCerts ConfigMap
- Argo CD cluster/repository Secrets
- Argo CD AppProjects CRs (custom resources)
- Argo CD Application CRs
- Argo CD ApplicationSet CRs

If you are using default settings (for example, for ConfigMaps) and you have another mechanism for backing these resources (for example, they are defined in Git or you backup your k8s cluster), then a backup would not necessarily be required (although this would of course be your own call, based on your own risk tolerance).

([Relevant code](https://github.com/argoproj/argo-cd/blob/master/cmd/argocd/commands/admin/backup.go))
